### ISSUE TITLE
Add Mastodon/Fediverse links in Social Media section

### ISSUE THREAD

**Issue by pReya** [tobimori/kirby-seo#92]
It would be very nice, if one could add Mastodon/Fediverse links in the social media section.

These links should then be rendered as `<link rel="me" href="https://whatever.com" />` in the head section, which will make the website appear as "verified" on the linked Mastodon profile (see: https://github.com/mastodon/mastodon/issues/10345).

I know, this is probably not SEO relevant, but it's similar in functionality to the other social media links, so I'd argue it should still be possible.

Are you open to a PR?

**Comment by tobimori** [comment]
yes 👏  actually something like this was my goal with this section

**Comment by nilshoerrmann** [comment]
Maybe related: https://blog.joinmastodon.org/2024/07/highlighting-journalism-on-mastodon/

**Comment by tobimori** [comment]
this is really cool, thanks for that.

**Comment by dewey** [comment]
Was just looking for that too, so just wanted to voice my support for that (Also not directly the Fediverse but Bluesky would also be nice to have).
